src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -45,26 +51,125 @@ object ConductrPlugin extends AutoPlugin {
     )
 
   override def projectSettings: Seq[Setting[_]] =
-    super.projectSettings ++ List(
+    List(
       // Here we try to detect what binary universe we exist inside, so we can
       // accurately grab artifact revisions.
       isSbtBuild := Keys.sbtPlugin.?.value.getOrElse(false) && (Keys.baseDirectory in ThisProject).value.getName == "project",
 
+      discoveredDist <<= (dist in Bundle).storeAs(discoveredDist).triggeredBy(dist in Bundle),
+      discoveredConfigDist <<= (dist in BundleConfiguration).storeAs(discoveredConfigDist).triggeredBy(dist in BundleConfiguration)
+    )
+
+  override def buildSettings: Seq[Setting[_]] =
+    List(
       sandbox := sandboxTask.value.evaluated,
       sandboxRunTaskInternal := sandboxRunTask(ScopeFilter(inAnyProject, inAnyConfiguration)).value,
       conduct := conductTask.value.evaluated,
```

Comment:
The above 3 are now `buildSettings` - makes sense as they apply to the build as a whole.